pyap_beauhurst/address.py

```diff
@@ -12,10 +12,23 @@
 
 
 class Address:
-    full_address: str
-    state: str
+    building_id: str
```

Comment:
Seems like this should be replaced with a dataclass and a validation function (or even pydantic)